jobs/nro-update/tests/test_nro_datapump.py

```diff
@@ -1,17 +1,15 @@
 from datetime import datetime
 
 import pytest
-import pytest_mock
 from pytz import timezone
-import pytz
 
 from nro.nro_datapump import nro_data_pump_update, create_expiry_date
 from namex.models import Request, Name, State, User
 
 
 expiry_date_test_data = [
     ('using epoch utc',            # test descriptive name
-     datetime.utcfromtimestamp(0), # start date - time
+     datetime(1970, 1, 1, 00, 00, tzinfo=timezone('US/Pacific', )), # start date - time
```

Comment:
What is the purpose of changing these dates from datetime.utcfromtimestamp(0) to the 1970 date? You've done it in a few places in this test file.